### ISSUE TITLE
CLI login not working

### ISSUE THREAD

**Issue by dazey3** [Bionus/imgbrd-grabber#2920]
**Bug description**

When running `grabber` from the command line, I sometimes get the warning:
```
[Warning] [JavaScript] 2 image(s) without URL found, login to view them
```

However, if I provide my username and password:
```
C:\[redacted]\Grabber_v7.10.1_x64> grabber-cli -f "C:\[redacted]\imgbrd-grabber\%artist%\%md5%.%ext%" -m 10000000 -t "[redacted]" -s "e621.net" -u "[redacted]" -w "[redacted]" --download
```
it seems to have no effect.

**Steps to reproduce**

Run `grabber-cli` with `-s "e621.net"` and using a tag that has images that are blocked unless you login. 

**System information**

- OS: Windows 11
- Grabber version: v7.10.1

**Comment by Bionus** [comment]
Yes, seems like the CLI parameters for login aren't mapped properly. Note that login does work when just going through the UI, which might be better in some cases since a source might have multiple login methods and more than two fields.

I'll make sure that the user/login config is mapped properly when passed, though.

**Comment by dazey3** [comment]
I know the UI is more popular, but I find the command line option much more accessible for my needs. It allows me to run batch scripts and mass backup several artists across several sites. I appreciate you taking the time to verify the issue and plan on fixing! Thanks so much!

**Comment by Bionus** [comment]
My point was that if you go through the UI, setup your login information, then go back to the CLI, you don't need to pass any login parameters. Definitely not great but you'd only need to use it to setup login.

**Comment by dazey3** [comment]
> My point was that if you go through the UI, setup your login information, then go back to the CLI, you don't need to pass any login parameters. Definitely not great but you'd only need to use it to setup login.

Oh, I see. That's a good tip! The login must be cached somewhere then... I'll give it a try until the fix can get in. Thanks!

**Comment by Bionus** [comment]
Yes, login information is stored in each site's config files.
